app/steps/ui/deceased/dod/index.js

```diff
@@ -14,7 +15,16 @@ class DeceasedDod extends DateStep {
     }
 
     handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        const dod = new Date(`${ctx.dod_year}-${ctx.dod_month}-${ctx.dod_day}'`);
```

Comment:
It looks like there is an additional `'` at the end of the date string

Reply:
That's removed